api/manager.py

```diff
@@ -13,33 +13,198 @@
 @bp.route("/orders", methods=["GET"])
 def orders():
     db = get_db()
+    error = None
+    query = """
+        SELECT *
+        FROM ordered_cone
+        """
+    orders = db.execute(query).fetchall()
+    print(orders)
     # get all orders, or a specific number
     # if we want to get only a few at a time, we may need to have this be a bit more involved
     # so that we don't end up returning the same few records over and over
     # this will be used in account management as well
 
+    if orders is None:
+        error = "Not implemented"
+    elif len(orders == 0):
+        error = "Empty table"
 
-@bp.route("/product", methods=["GET"])
+    if error:
+        return json.dumps({"error": error})
+    else:
+        return json.dumps({"orders": orders})
+
+
+@bp.route("/product", methods=["GET", "PUT", "POST"])
 def accounting():
     if request.method == "GET":
         db = get_db()
+        error = None
         # get info on one specific product
+        query = """
+            SELECT display_name, stock, price_per_unit
+            FROM product
+            WHERE id = {search}
+            """
+        # Idea is each item from table will be grabbed incrementally
+        item = db.execute(query).fetchone()
+        print(item)
+
+        if item is None:
+            error = "Not implemented"
+        elif len(item == 0):
+            error = "Empty table"
+
+        if error:
+            return json.dumps({"error": error})
+        else:
+            return json.dumps({"item": item})
 
     if request.method == "PUT":
+        # grab data
+        data = request.get_json()
+        displayName = data["display_name"]
+        stock = data["stock"]
+        pricePerUnit = data["price_per_unit"]
+        productType = data["product_type"]
```

Comment:
are you 100% sure this will work? iirc if you try to pull data from the request and it doesn't exist, the function will crash rather than assigning null, etc. and the put request will specifically have the information that needs to be changed

if youve tested thoroughly and are certain it works i trust u but i wanna get confirmation lol